terraform/environments/mlra/modules/ecs/main.tf

```diff
@@ -361,13 +381,13 @@ resource "aws_ecs_service" "ecs_service" {
   }
 
   load_balancer {
-    target_group_arn = data.aws_lb_target_group.target_group.id
+    target_group_arn = var.lb_tg_arn
     container_name   = var.app_name
     container_port   = var.server_port
   }
 
   depends_on = [
-    aws_iam_role_policy_attachment.ecs_task_execution_role, aws_ecs_task_definition.windows_ecs_task_definition, aws_ecs_task_definition.linux_ecs_task_definition
+    aws_iam_role_policy_attachment.ecs_task_execution_role, aws_ecs_task_definition.windows_ecs_task_definition, aws_ecs_task_definition.linux_ecs_task_definition, aws_cloudwatch_log_group.cloudwatch_group
   ]
```

Comment:
The ECS instance depends on the CloudWatch Log Group to send logs to, where config is defined in the task definition